src/excalidraw-app/data/firebase.ts

```diff
@@ -106,7 +106,7 @@ const encryptElements = async (
   key: string,
   elements: readonly ExcalidrawElement[],
 ): Promise<{ ciphertext: ArrayBuffer; iv: Uint8Array }> => {
-  const json = JSON.stringify(elements);
+  const json = JSON.stringify(elements.filter((el) => !el.isDeleted));
```

Comment:
You can use `getNonDeletedElements(elements)` instead

Reply:
To retain recently deleted elements like @dwelle requested, I kept the `!el.isDeleted` way.